frontend/packages/knative-plugin/src/components/overview/EventSinkServicesOverviewList.tsx

```diff
@@ -60,7 +60,7 @@ const EventSinkServicesOverviewList: React.FC<EventSinkServicesOverviewListProps
       ) : (
         <span className="text-muted">No services found for this resource.</span>
       )}
-      {pods?.length && <PodsOverview pods={pods} obj={obj} allPodsLink={linkUrl} />}
+      {pods?.length > 0 && <PodsOverview pods={pods} obj={obj} allPodsLink={linkUrl} />}
```

Comment:
If pods length is 0, don't we want to show empty msg? `No pods found for this resource`

Reply:
yeah some sources may not have Pods or associated deployments so had this check